### PR TITLE
OCPBUGS#7489: Webhook OpenShift timeout cannot be increased above 13s

### PR DESCRIPTION
**Pull request by xenolinux**
As per reporter's feedback this PR adds some minor changes in the wordings

Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-7489

Link to docs preview:  https://56441--docspreview.netlify.app/openshift-enterprise/latest/architecture/admission-plug-ins.html#admission-webhooks-about_admission-plug-ins

QE review: @gangwgr 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Amrita42** [comment]
lgtm

File: modules/admission-webhooks-about.adoc [hunk 1]
```diff
@@ -48,5 +48,5 @@ Some common webhook admission plugin use cases include:
 
 [NOTE]
 ====
-The default webhook timeout value in {product-title} is 13 seconds and cannot be changed.
+The default webhook highest timeout value in {product-title} is 13 seconds and cannot be increased.
```
Comment (snarayan-redhat) [inline modules/admission-webhooks-about.adoc:51]:
```suggestion
The maximum default webhook timeout value in {product-title} is 13 seconds, and it cannot be changed.
```

**Review by snarayan-redhat — COMMENTED** [review]
Just one nit. Otherwise LGTM. Will merge once the change is pushed.

File: modules/admission-webhooks-about.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/admission-webhooks-about.adoc]: Addressed

**Pull request merged**